pandas/tests/test_panel.py

```diff
@@ -1560,6 +1560,17 @@ def test_truncate_fillna_bug(self):
         # it works!
         result.fillna(value=0.0)
 
+    def test_truncate_outofbounds(self):
```

Comment:
I got you. I ran into issues (memory, indeces) when trying to generalize, so decided to move it under Series (only).